### PR TITLE
Added Common Core Standards and multiple targets.

### PR DESCRIPTION
**Pull request by agorina**
The requirements for this change is noted on this page in "Changes to Common Core standards and Claim/Targets " section: https://confluence.fairwaytech.com/pages/viewpage.action?pageId=48824335

This review changes the schema and loads the codes.
All other changes will be in the Ingest and Migrate.

### CODE REVIEW

File: warehouse/sql/V201702061486427077__initial_ddl.sql [hunk 1]
```diff
@@ -190,10 +199,26 @@ CREATE TABLE IF NOT EXISTS item (
   CONSTRAINT fk__item__claim FOREIGN KEY (claim_id) REFERENCES claim(id),
   CONSTRAINT fk__item__target FOREIGN KEY (target_id) REFERENCES target(id),
   CONSTRAINT fk__item__asmt FOREIGN KEY (asmt_id) REFERENCES asmt(id),
-  CONSTRAINT fk__item__math_practice FOREIGN KEY (math_practice) REFERENCES math_practice(practice), 
+  CONSTRAINT fk__item__math_practice FOREIGN KEY (math_practice) REFERENCES math_practice(practice),
   CONSTRAINT fk__item__dok FOREIGN KEY (dok_id) REFERENCES depth_of_knowledge(id)
 );
 
+CREATE TABLE IF NOT EXISTS item_other_target (
```
Comment (agorina) [inline warehouse/sql/V201702061486427077__initial_ddl.sql]: Any suggestions for a better name? Or is it good enough?

File: warehouse/sql/V201702061486427077__initial_ddl.sql [hunk 2]
```diff
@@ -157,11 +157,20 @@ CREATE TABLE IF NOT EXISTS subject_claim_score (
...
   claim_id smallint NOT NULL,
+  natural_id varchar(20) NOT NULL,
   code varchar(10) NOT NULL,
   description varchar(500) NOT NULL,
   CONSTRAINT fk__target__claim FOREIGN KEY (claim_id) REFERENCES claim(id)
 );
 
+CREATE TABLE IF NOT EXISTS common_core_standard (
+  id smallint NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  subject_id tinyint NOT NULL,
```
Comment (malaffoon) [inline warehouse/sql/V201702061486427077__initial_ddl.sql]: super minor nit: by putting the subject_id before natural_id, it makes the reader (well, me) think that subject_id is somehow part of the discriminator for the entry. Which it isn't, it is just an attribute for the entry. Upshot: it makes more sense to me to have `id`, then `natural_id` then all the non-identifying attributes (ditto for target table).

File: warehouse/sql/V201702061486427077__initial_ddl.sql (hunk 1 above)
Comment (malaffoon) [inline warehouse/sql/V201702061486427077__initial_ddl.sql]: good enough

File: warehouse/sql/V201702061486427077__initial_ddl.sql (hunk 2 above)
Comment (agorina) [inline warehouse/sql/V201702061486427077__initial_ddl.sql]: Will change.

**Pull request merged**